engine/gameobject/src/dmsdk/gameobject/gameobject.h

```diff
@@ -362,6 +362,8 @@ namespace dmGameObject
         uint8_t  m_IsGamepad : 1;
         uint8_t  m_GamepadDisconnected : 1;
         uint8_t  m_GamepadConnected : 1;
+        dmHID::GamepadPacket m_GamepadPacket;
+        uint32_t m_HasGamepadPacket;
```

Comment:
Same comment as below, don't break up bit patterns, and use the current coding style (use a bit pattern for the boolean)